### PR TITLE
Disable wildcard for tensorflow model attributes to detect input errorr earlier

### PR DESCRIPTION
**Pull request by shendiaomo**
```sql
SELECT * FROM iris.train TO TRAIN DNNClassifier WITH model.n_class=3, model.hidden_units=[20, 20] LABEL class INTO iris.dnn_model;
```
The above statement will throw a very long error message.
After this change, the error message is
```
2020/03/10 11:45:05 runSQLProgram error: unsupported attribute model.n_class
```

### CODE REVIEW

**Comment by typhoonzero** [comment]
Keras models accept self-defined model parameters which may not know in advance, you need to check whether it's a Keras model and do the model parameter check.

**Comment by shendiaomo** [comment]
> Keras models accept self-defined model parameters which may not know in advance, you need to check whether it's a Keras model and do the model parameter check.

Roger that.

File: pkg/sql/codegen/tensorflow/codegen.go [hunk 1]
```diff
@@ -307,6 +306,17 @@ func InitializeAttributes(trainStmt *ir.TrainStmt) error {
 	constructOptimizers(trainStmt)
 	constructLosses(trainStmt)
 	attrValidator := modelAttr.Update(commonAttributes)
+	if len(modelAttr) == 0 {
```
Comment (typhoonzero) [inline pkg/sql/codegen/tensorflow/codegen.go:309]: The variable name `modelAttr` makes this function hard to read, maybe can call it `expectedModelAttrs`?

**Review by typhoonzero — APPROVED** [review]
LGTM with a naming comment.

File: pkg/sql/codegen/tensorflow/codegen.go (hunk 1 above)
Comment (shendiaomo) [reply pkg/sql/codegen/tensorflow/codegen.go:309]: Fix later.

**Pull request merged**